editor/app/crashes/[record_locator]/page.tsx

```diff
@@ -124,7 +133,7 @@ export default function CrashDetailsPage({
           />
         </Col>
       </Row>
-      <Row>
+      <Row id="A">
```

Comment:
is this a weird way to do this with the element id just being "A" "U" or "P"? :shrug: